tower-http/src/follow_redirect/mod.rs

```diff
@@ -0,0 +1,325 @@
+//! Middleware for following redirections.
```

Comment:
A few high level examples here would be nice. I think there should be one showing the default use case using `FollowRedirectLayer::standard` and one that customizes the policy in some way.

Something like what I've done here https://github.com/tower-rs/tower-http/pull/68.

Reply:
Added in ac8818b02904d0055f1fd2a8c0013c7e5c7e7f80.

Reply:
Awesome!